modules/osd-cluster-wide-proxy.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * networking/proxies/configuring-cluster-wide-proxy.adoc
+
+[id="osd-cluster-wide-proxy-config_{context}"]
+= Configuring a cluster-wide proxy using {product-title}
+
+You can add a proxy to your cluster when you are creating a new one. Prior to installation, however, you should verify that the proxy is accessible from the intended cluster virtual private cloud (VPC) and its private subnets.
+
+.Procedure
+To create a cluster with a proxy, run the following command:
+[source, terminal]
+----
+ocm create cluster \
+ <other arguments here> \
+ --additional-trust-bundle-file <path-to-CA-bundle-file> \
+ --http-proxy http://<username>:<pswd>@<ip>:<port> \
+ --https-proxy http(s)://<username>:<pswd>@<ip>:<port>
+----
+
+[id="osd-proxy-considerations"]
+== Considerations
+
+1. The `http-proxy`, `https-proxy`, and `additional-trust-bundle-file` arguments are all optional.
+1. If you use the `additional-trust-bundle-file` option without an `http(s)-proxy` argument, the passed additional trust bundle will be set on the cluster, but it will not be configured to be used with the proxy.
+1. The `additional-trust-bundle-file` argument is a file path pointing to a bundle of Privacy Enhanced Mail (PEM)-encoded X.509 certificates, which are all concatenated together. The `additionalTrustBundle` field is required unless
+the proxy's identity certificate is signed by an authority from the {op-system} trust
+bundle.
```

Comment:
I'd fix these so that you don't have any hard wraps (sentences should all be on a single line)